util/chplenv/third_party_utils.py

```diff
@@ -15,10 +15,13 @@
 #
 @memoize
 def default_uniq_cfg_path():
-    return '{0}-{1}-{2}'.format(chpl_platform.get('target'),
-                                chpl_compiler.get('target'),
-                                chpl_arch.get('target', map_to_compiler=True,
-                                         get_lcd=using_chapel_module()).arch)
+    arch_val = chpl_arch.get('target',
+                             map_to_compiler=True,
```

Comment:
Don't think this should be on a separate line.

Reply:
just `map_to_compiler=True, ` doesn't need to be on a new line. I'm fine with arch_val getting pulled out though

Reply:
Just in case that's still not clear, all I'm saying is I'd write this as:

```python
    arch_val = chpl_arch.get('target', map_to_compiler=True,
                             get_lcd=using_chapel_module()).arch
```